app/controllers/application_controller.rb

```diff
@@ -10,11 +10,6 @@
 class ApplicationController < ActionController::Base
   include Vmdb::Logging
 
-  if Vmdb::Application.config.action_controller.allow_forgery_protection
-    db = MiqDatabase.connected? ? MiqDatabase.first : nil
-    protect_from_forgery :secret => db ? db.csrf_secret_token : SecureRandom.hex(64), :except => :csp_report, :with => :exception
-  end
-
```

Comment:
I tested it manually and this change breaks it.  We need to call `protect_from_forgery` in the controllers we need the protection enforced.  config.action_controller.allow_forgery_protection=true in production doesn't enable the verify_authenticity_token callback.   I'll fix this.  But good question, I'm not sure how to test this since ` config.action_controller.allow_forgery_protection=false` in test mode.